www/pfSense-pkg-vHosts/files/usr/local/www/vhosts_certs.php

```diff
@@ -0,0 +1,465 @@
+<?php
```

Comment:
We have a facility that lets packages tell the Certificate Manager that they are in use. For example, here is the code that does it in the ACME package:

https://github.com/pfsense/FreeBSD-ports/blob/devel/security/pfSense-pkg-acme/files/usr/local/pkg/acme.xml#L39

https://github.com/pfsense/FreeBSD-ports/blob/devel/security/pfSense-pkg-acme/files/usr/local/pkg/acme/acme.inc#L23